Dockerfile.semantic-release

```diff
@@ -1,13 +1,5 @@
-FROM jfloff/alpine-python:2.7-slim
-RUN apk update
-RUN apk add --no-cache bash git
-RUN /entrypoint.sh \
-    -a git \
-    -p python-semantic-release==3.11.2 \
-    && echo
-RUN apk add pcre git
+FROM 997386117669.dkr.ecr.us-east-1.amazonaws.com/semantic-release:py3
```

Comment:
where is this initial image from/how does it get created?

Reply:
Sorry, it looks like I forgot to summarize on the ticket where I ended up. Here's what I did:
* Started a container locally from the existing image (`jfloff/alpine-python:2.7-slim`)
* Installed Python 3 and all the stuff needed to make that work (see ticket for details)
* Generated a new image from that container
* Pushed that image to [semantic-release ECR](https://us-east-1.console.aws.amazon.com/ecr/repositories/semantic-release/?region=us-east-1) with a `py3` tag.
* Tagged previous image `py2`

Reply:
(Added that to the ticket for good measure)

Reply:
Thanks for asking the question. Though semver isn't exactly a mission critical part of our stack, It would be bad if we were relying on a "kitten" image with no record of how it was built.